### PR TITLE
Remove eval call for running rest client command

### PR DESCRIPTION
**Pull request by lmcglohon**

### CODE REVIEW

**Comment by mejackreed** [comment]
Yes please.. !

**Comment by mejackreed** [comment]
Heads up.. I just rebased this and force pushed it.

File: lib/discovery_dispatcher/indexing_job.rb [hunk 1]
```diff
@@ -10,42 +10,47 @@ def perform
...
       response = nil
       begin
-        response = eval(request_command)
+        conn = Faraday.new(url: request_url)
+        if type == 'index'
+          response = conn.put do |request|
+            request.body = ''
+          end
+        else
+          response = conn.delete
```
Comment (mejackreed) [inline lib/discovery_dispatcher/indexing_job.rb:42]: I'm not exactly sure what case this is handling. Can you expound a bit?

Comment (lmcglohon) [inline lib/discovery_dispatcher/indexing_job.rb:42]: This is the API call for the sw-indexer.  If the call is for indexing (in the case of new druid or updated druid), the API call is a PUT with the appropriate url which is built in build_request_url. The PUT requires a payload called body which is empty in our scenario. If the call is for a deleted druid, the API call is a DELETE with the appropriate URL from build_request_url. Does that make sense?
@mejackreed

Comment (mejackreed) [inline lib/discovery_dispatcher/indexing_job.rb:42]: Sounds good.. .. added an issue adding a test #32

**Pull request merged**